### PR TITLE
adds configuration files and fixes multipart upload

### PR DESCRIPTION
**Pull request by marcvivancos**

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      - name: Download coverage report
+        uses: actions/download-artifact@v1
+        with:
+          name: ubuntu-latest_coverage
+
+      - name: Analyze with SonarCloud
+        uses: sonarsource/sonarcloud-github-action@master
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
```
Comment (guumaster) [inline .github/workflows/tests.yml:61]: Have you already created this token and project in SonarCube?

File: README.md [hunk 2]
```diff
@@ -13,16 +13,18 @@ Low-level GraphQL client for Go.
 Make sure you have a working Go environment. To install graphql, simply run:
```
Comment (guumaster) [inline README.md:13]: Add a note to the original repo and the reason to maintain a new fork. The original  seems unmaintained

File: request.go [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+func NewRequest(q string) *Request {
+	req := &Request{
+		q:      q,
+		Header: make(map[string][]string),
+	}
+
+	return req
+}
+
+// getOperationName parses operation name from query.
```
Comment (guumaster) [inline request.go:32]: the doc comment should be `OperationName`

File: .github/workflows/tests.yml (hunk 1 above)
Comment (angelsanchez) [reply .github/workflows/tests.yml:61]: yes

**Pull request merged**